src/tasks/dev_prerun_hermes.js

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright: Ambrosus Technologies GmbH
+Email: tech@ambrosus.com
+
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+This Source Code Form is “Incompatible With Secondary Licenses”, as defined by the Mozilla Public License, v. 2.0.
+*/
+
+import {WinstonConsoleLogger} from '../utils/loggers';
+import Builder from '../builder';
+import config from '../../config/config';
+import {addToKycWhitelist, registerAdminAccount} from './dev_prerun';
+
+async function onboardAsHermes(dataModelEngine, logger) {
```

Comment:
Maybe move this to the same place where `addToKycWhitelist` and `registerAdminAccount` are?

Reply:
This one is specific for hermes

Reply:
But it can be used somehere else for test setup purposes.
Task script will be then just gathering proper stuff and running it, without looking into details.